bsp/esp32_s3_usb_otg/idf_component.yml

```diff
@@ -1,4 +1,4 @@
-version: "1.6.2"
+version: "1.6.3"
```

Comment:
This is not right file for bump BSP version. You should do it inside `esp_bsp_generic`

Reply:
Sorry for inconvenience. I have reverted this file and bumped this inside the **esp_bsp_generic** directory.

Reply:
Thank you :-)